CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Go to [legend](#legend---types-of-changes) for further information about the types of changes.
 
+## [Unreleased]
+
+### Changed
+
+- _WD_ElementSelectAction
+	- Added `singleSelect` command
+	- Revised columns returned by `options` and `selectedOptions` commands
+	- Refactored Javascript coding
```

Comment:
Yes, the CL needs some additional updates before merging. However, I think it is unrealistic to to expect that it will contain a list of every single minor change that has occurred.

Reply:
> However, I think it is unrealistic to to expect that it will contain a list of every single minor change that has occurred.

I just was unsure if you forget to mention or just intentionally do not want to mention to keep release not small.